.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,52 @@
+name: reusable_testing
```

Comment:
Why 'reusable' if it is an integration testing?

Reply:
Sure, it is more like a template with 'build expirationd and run against given tarantool artifact' steps rather than a real workflow. We agreed on this name with Yaroslav during work on https://github.com/tarantool/vshard/pull/299. I had a wish to have something like 'reusable' or 'callable' in the name and we stick on the former, because GitHub's feature itself is called 'reusable workflows'.

We agreed on the 'integration' name for the caller workflow in tarantool (because it is about integration testing and because 'integration_testing' is too long for GitHub UI).

Reply:
Probably if we chose `callable` instead of `reusable`, maybe it would confuse people less. What do you think?

Reply:
Okay, let's keep the name of workflow as is. Anyway, we already use the same name in other external repositories, and it would be good to make it align with them.